testie.wren

```diff
@@ -25,13 +25,27 @@ class Testie {
         _skips = []
         _name = name
         _fails = 0
+        _abortAfterFailures = true
         _afterEach = _beforeEach = Fn.new {}
         fn.call(this, Skipper.new(this))
     }
     static test(name, fn) { Testie.new(name,fn).run() }
+    static test(name, options, fn) {
+        var testie = Testie.new(name,fn)
+        for (option in options) {
+            if (option.key == "reporter") {
+                testie.reporter = option.value
+            }
+            if (option.key == "abortAfterFailures") {
+                testie.abortAfterFailures_ = option.value
+            }
+        }
+        testie.run()
+    }
     expect(v) { Expect.that(v) }
     afterEach(fn) { _afterEach = fn }
     beforeEach(fn) { _beforeEach = fn }
+    abortAfterFailures_=(v){ _abortAfterFailures = v }
```

Comment:
What is the logic to a private accessor vs just using `_abortAfterFailures` directly?

Reply:
Because I expose that in `Testie.test("name" {"abortAfterFailures": false}) {...}` so I can't access the field directly.

Reply:
Ah.  The problem is where the code is located (IMHO)... the option "hash" should be passed directly into `new`... (new needs to support that anyways as it's a perfectly valid way to setup a test without running it)... the static method should just be a very thin wrapper. And then `new` does have access to the fields, of course.